src/components/layout/productCard/ProductCard.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Slider from 'react-slick';
+import heart from '../../../assets/svgs/heart.svg';
+import { Col } from 'react-bootstrap';
+import './style.scss';
+import heartBroken from '../../../assets/svgs/heart-broken.svg';
+
+export default function ProductCard(props) {
+	const [isLiked, setIsLiked] = React.useState(false);
+
+	const settings = {
+		className: 'silderImages',
```

Comment:
indentation still wrong please fix and i'll review this PR, write PTAL when ready